lib/portage/util/_urlopen.py

```diff
@@ -29,7 +29,7 @@ def have_pep_476():
 def urlopen(url, if_modified_since=None, headers={}, proxies=None):
```

Comment:
How about if we add a default `timeout=10` parameter up here?

Reply:
Done.